tests/data/instances/sequence_tagging/verb_semantics_instance_test.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=no-self-use,invalid-name
+from typing import List
+
+from numpy.testing import assert_array_almost_equal
+
+from deep_qa.data.data_indexer import DataIndexer
+from deep_qa.data.instances.instance import TextInstance
+from deep_qa.data.instances.sequence_tagging.pretokenized_tagging_instance import PreTokenizedTaggingInstance
+from deep_qa.data.instances.sequence_tagging.verb_semantics_instance import VerbSemanticsInstance
+from deep_qa.data.tokenizers import tokenizers
+from deep_qa.common.params import Params
+from ....common.test_case import DeepQaTestCase
+
+
+class TestVerbSemanticsInstance(DeepQaTestCase):
+    def setUp(self):
```

Comment:
If you aren't adding any functionality to a superclass, there's no need to implement it, so you can just remove setUp and tearDown - if you look in DeepQaTestCase they are already implemented.